app/controllers/qsets_controller.rb

```diff
@@ -3,17 +3,25 @@ class QsetsController < ApplicationController
 
   # shows all qsets
```

Comment:
This comment should either be updated or removed, it doesn't appear to be accurate anymore.